poms/bom-deployment/pom.xml

```diff
@@ -230,6 +230,11 @@
                 <artifactId>camel-quarkus-tarfile-deployment</artifactId>
                 <version>${camel-quarkus.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel.quarkus</groupId>
+                <artifactId>camel-quarkus-fhir-deployment</artifactId>
+                <version>${project.version}</version>
```

Comment:
Please use `${camel-quarkus.version}` instead of `${project.version}`.

Reply:
Done